common/extensions/api/_api_features.json

```diff
@@ -44,6 +44,30 @@
       "chrome://newtab/*"
     ]
   }],
+  "tabs": [{
+    "channel": "stable",
+    "extension_types": ["extension", "legacy_packaged_app"],
+    "contexts": ["blessed_extension"],
+    "default_parent": true
+  }, {
+    "channel": "stable",
+    "contexts": ["webui"],
+    "matches": [
+      "chrome://bookmarks/*",
+      "chrome://read-later.top-chrome/*",
+      "chrome://tab-strip/*",
+      "chrome://wallet-panel.top-chrome/*",
+      "chrome://wallet/*"
+    ]
+  }, {
+    "channel": "stable",
+    "contexts": ["webui_untrusted"],
+    "matches": [
+      "chrome-untrusted://crosh/*",
+      "chrome-untrusted://terminal/*"
+    ],
+    "platforms": ["chromeos"]
```

Comment:
just wondering: do we need to copy everything?

Reply:
fairly sure, I don't like this way of patching it though, but I don't think we have an alternative. Maybe patching directly is better.

Reply:
yeah, I'm asking because our version of `"bookmarks"` is already behind chromium